backend/jsonProcessor.js

```diff
@@ -34,8 +34,10 @@ async function processJsonFile() {
           ? business.types[0].trim()
           : "Uncategorized";
         const businessHours = business.opening_hours
-          ? business.opening_hours.weekday_text || ["Unknown"]
-          : ["Unknown"];
+          ? business.opening_hours.weekday_text || ["Not available"]
+          : ["Not available"];
+        const userRatingsTotal = business.user_ratings_total || 0;
```

Comment:
I think this will return a boolean every time. You should use null coalescence instead. `business.user_ratings_total ?? 0`